src/tutorial/fem.md

```diff
@@ -113,20 +113,21 @@ To learn more, you can visit MFEM's <a href="../../fem/">Finite Element Method</
 
 ### <i class="fa fa-check-square-o"></i>&nbsp; Annotated Example 1
 
-MFEM's Example 1 implements the above simple finite element method for the Poisson problem
+MFEM's Example 1 implements the above simple FEM for the Poisson problem
 in the source file [examples/ex1.cpp](https://github.com/mfem/mfem/blob/master/examples/ex1.cpp).
 We set $f=1$ in (1) and enforce homogeneous Dirichlet boundary conditions on the whole boundary.
 
-Below we highlight selected portion of the example code and connect them with
+Below we highlight selected portions of the example code and connect them with
 the description in the previous section. You can follow along by browsing
-`ex1.cpp` in your VS Code browser window.
+`ex1.cpp` in your VS Code browser window. In the settings of this tutorial, the
+visualization will automatically update in the GLVis browser window.
```

Comment:
I moved this statement b/c it makes me think I'm supposed to see a change in the example viz, but at that point I haven't entered any new commands.

Reply:
Thanks!